pom.xml

```diff
@@ -1464,6 +1464,9 @@
             <goals>
               <goal>sort</goal>
             </goals>
+            <!-- Empty phase to skip automatically running this as part of a build process -->
```

Comment:
The `execution` section has an ID `manual-sort`, so that one can invoke the plugin with configurations set in this particular section with `mvn sortpom:sort@manual-sort`. If the execution ID is omitted, the plugin is invoked with hard-coded default setting of the plugin.

So I think leaving an `execution` section with an empty phase serves two purposes: to remind people some useful plugin usages, and to provide good default settings.

Reply:
got it, it is required to define the section for the command to operate correctly. then it is definitely necessary